site/docs/master/README.md

```diff
@@ -1,5 +1,3 @@
-[![Build Status][1]][2] [![Go Report Card][3]][4] ![GitHub release][5] [![License][6]][7]
```

Comment:
Looks like you can remove links [1]-[7] from the footer as well.

Reply:
Ahh good call!